app/assets/javascripts/utility.js

```diff
@@ -55,14 +55,18 @@
   };
 
   microp.format.removeGitPrefix = function(str) {
-    return str.split('https://github.com/').join('');
+    return str.includes('https')
```

Comment:
@adelevie for sure. @vzvenyach recently introduced me to https://github.com/18F/micropurchasedata. There isn't much too many methods there right now, but a lot of these methods could be moved there. I created #480 for this is